electron/actions/app/checkForUpdates.js

```diff
@@ -119,9 +117,24 @@ function checkWithUpdate () {
 }
 
 export default function checkForUpdates () {
-  if (isMacos) {
-    checkWithoutUpdate()
-  } else {
+  const isAutoupdateAvailable =
+    getElectronStoreKey(
+      'updates.isAutoupdateAvailable'
+    )
+
+  const isAutoupdateEnabled =
+    getElectronStoreKey(
+      'updates.isAutoupdateEnabled'
+    )
+
+  const isAutoupdate = (
```

Comment:
Suggest changing that to `willAutoupdate`

Reply:
Sorry, all my boolean variables start with `is`.